### ISSUE TITLE
LDK 3.0.1 failed with SyntaxError: loop: Line 5:2549 Unexpected token

### ISSUE THREAD

**Issue by hans-hsieh-asus** [open-olive/loop-development-kit#241]
Hi, if I upgrade to LDK 3.0.1, it looks like the loop will fail with the error below:
```
2021-06-22T18:30:57+08:00 TRC Sending event (with base values added) event={"action":"UI Event Handlers","category":"Errors","documentEncoding":"","documentHostname":"","documentLocationUrl":"","documentPath":"","label":"failed to restart secure loop: failed to start loop: failed to run loop: failed to compile loop: SyntaxError: loop: Line 5:2549 Unexpected token { (and 1231 more errors)","screenColorDepth":"","screenResolution":"","userAgent":"","userLanguage":"","viewportSize":""} system=analytics
2021-06-22T18:30:57+08:00 ERR message received with name newError but no handler is registered for that name system=UI
2021-06-22T18:30:57+08:00 ERR map[] system=UI
```
The error seems to be unrelated to the code of the loop.
If I use the sample code [here](https://github.com/open-olive/loop-development-kit/tree/main/ldk/javascript/examples/javascript-loop) directly, it works. But if I change [this line](https://github.com/open-olive/loop-development-kit/blob/e993cc802294e47ed2e271aa7052a2999f15f748/ldk/javascript/examples/javascript-loop/package.json#L19) to `"@oliveai/ldk": "^3.0.1"`, it will also have the same problem.

**Comment by epiccoleman** [comment]
Hello Hans. I was able to reproduce this issue locally. I'll need to do some more investigation to figure out what's going on. Thanks for bringing this to our attention.

**Comment by dmorrison-olive** [comment]
I also get this error when I updated a loop from `3.0.0` to `3.0.1`.

**Comment by epiccoleman** [comment]
Hello again @hans-hsieh-asus . After some investigation today we discovered a bug in LDK 3.0.1 that was causing the issue. We have taken down the 3.0.1 release and published 3.0.2, which fixes the issue. 

When you have some time, please try updating to 3.0.2, and let us know if you encounter any issues. Thanks again for bring this to our attention.

**Comment by epiccoleman** [comment]
Closing this issue for now, since this error should be fixed in 3.0.2. Please feel free to re-open if you continue to experience this issue after updating to 3.0.2.
